packages/calcite-components/src/components/fab/fab.scss

```diff
@@ -1,11 +1,22 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-fab-background-color: Specifies the component's background color.
+ * @prop --calcite-fab-shadow: Specifies the component's shadow.
+ */
+
 :host {
-  @apply flex bg-transparent;
+  @apply flex;
+
+  background-color: var(--calcite-fab-background-color, transparent);
 }
 
 @include disabled();
 
 calcite-button {
-  @apply shadow-2;
+  box-shadow: var(--calcite-fab-shadow, 0 6px 20px -4px rgba(0, 0, 0, 0.1), 0 4px 12px -2px rgba(0, 0, 0, 0.08));
```

Comment:
I think you'd need to set this as an internal prop and accept an override - if a user were to override this, the other two hardcoded values would still appear, correct?